Makefile

```diff
@@ -1,6 +1,20 @@
+FFMPEG_PREFIX ?= $(shell pwd)/ffmpeg/$(shell uname -s)-$(shell uname -m)
+FFMPEG_OPTS ?= --prefix=$(FFMPEG_PREFIX) \
+               --enable-static \
+               --disable-shared \
+               --disable-programs \
+               --disable-doc \
+               --disable-everything \
+               --enable-decoder=h264 \
+               --enable-decoder=hevc \
+               --enable-swscale
 
-bin/viamrtsp: *.go cmd/module/*.go
-	go build -o bin/viamrtsp cmd/module/cmd.go
+.PHONY: build-ffmpeg
```

Comment:
there are lots of other `.PHONY` make targets here, such as `test`, `lint`, `updaterdk`, etc

Reply:
Good call - added the complete phony target list here.